### PR TITLE
rtm: migrate to Conan v2

### PR DESCRIPTION
**Pull request by valgur**
Specify library name and version:  **lib/1.0**

---

- [ ] I've read the [contributing guidelines](https://github.com/conan-io/conan-center-index/blob/master/CONTRIBUTING.md).
- [ ] I've used a [recent](https://github.com/conan-io/conan/releases/latest) Conan client version close to the [currently deployed](https://github.com/conan-io/conan-center-index/blob/master/.c3i/config_v1.yml#L6).
- [ ] I've tried at least one configuration locally with the [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

File: recipes/rtm/all/test_package/test_package.cpp [hunk 1]
```diff
@@ -1,3 +1,6 @@
+// workaround for a missing include in rtm
+#include <limits>
```
Comment (RubenRBS) [inline recipes/rtm/all/test_package/test_package.cpp:2]: Is this a bug in the code itself? Has it been fixed in any newer version/reported upstream?

**Review by RubenRBS — COMMENTED** [review]
Thanks a lot! I have one question about the change to the test package code that suprised me, otherwise looks good :)

File: recipes/rtm/all/test_package/test_package.cpp (hunk 1 above)
Comment (valgur) [reply recipes/rtm/all/test_package/test_package.cpp:2]: It's a bug in the project itself, yes. I have not created a bug repirt or PR yet, though.

**Review by RubenRBS — APPROVED** [review]
Approving now, thanks a lot! (If you have the time, feel free to report the issue upstream, I'm sure they'll appreciate it!)

**Review by czoido — APPROVED** [review]

**Pull request merged**